vendor/abeeway/abeeway-compact-tracker.yaml

```diff
@@ -2,16 +2,15 @@ name: Compact Tracker
 description: The Abeeway Compact Tracker is a highly versatile multi-mode tracker for outdoor and indoor geolocation with embedded sensors combining GPS, Low-power GPS, Wi-Fi Sniffer, BLE, and LoRa TDoA geolocation technologies.
 # Hardware versions (optional, use when you have revisions)
 hardwareVersions:
-  - version: '1.0'
-    numeric: 1
+  - version: '3.4'
 
 # Firmware versions (at least one is mandatory)
 firmwareVersions:
   - # Firmware version
-    version: '2.1'
+    version: '2.6.1'
     numeric: 2
     hardwareVersions:
-      - '1.0'
+      - '3.4'
```

Comment:
Hardware and firmware versions cannot be retracted. This will break devices already created with these versions.